repos.d/rpm/openmandriva.yaml

```diff
@@ -0,0 +1,42 @@
+###########################################################################
+# OpenMandriva
+###########################################################################
+- name: openmandriva_3
+  type: repository
+  desc: OpenMandriva 3.x
+  family: openmandriva
+  color: '045e9d'
+  sources:
+      # XXX: */backports_testing and */updates_testing not included, are these needed?
+    - name: [ main/release, main/updates, main/testing, non-free/release, non-free/updates, non-free/testing, restricted/release, restricted/updates, restricted/testing ]
+      fetcher: Repodata
+      parser: Repodata
+      url: 'http://abf-downloads.openmandriva.org/3.0/repository/SRPMS/{source}/'
+      subrepo: '{source}'
+  repolinks:
+    - desc: OpenMandriva home
+      url: http://openmandriva.org/
+  packagelinks:
+    - desc: Package repository
+      url: 'https://github.com/OpenMandrivaAssociation/{name}'
+  tags: [ all, production, openmandriva, rpm ]
+
+- name: openmandriva_cooker
+  type: repository
+  desc: OpenMandriva Cooker
+  family: openmandriva
+  color: '045e9d'
+  sources:
+      # XXX: */backports_testing and */updates_testing not included, are these needed?
+    - name: [ main/release, main/updates, main/testing, non-free/release, non-free/updates, non-free/testing, restricted/release, restricted/updates, restricted/testing ]
```

Comment:
Most of these subrepos are not there, there's only */release. Same question for contrib repo.